### PR TITLE
#1489 add surface area to volume ratio factor

### LINKED ISSUE #1489: Multiply by surface area to volume ratio
In all the degradation models that depend on a current (d.../dt = j), the RHS should be multiplied by surface area to volume ratio (a) to account for possible changes in active material

### PR DESCRIPTION
**Pull request by tinosulzer**
# Description

Add surface area to volume ratio factor

Fixes #1489 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: examples/scripts/compare_lithium_ion.py [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 # load models
 models = [
-    pybamm.lithium_ion.SPM(),
-    pybamm.lithium_ion.SPMe(),
+    # pybamm.lithium_ion.SPM(),
+    # pybamm.lithium_ion.SPMe(),
     pybamm.lithium_ion.DFN(),
-    pybamm.lithium_ion.NewmanTobias(),
+    # pybamm.lithium_ion.NewmanTobias(),
```
Comment (brosaplanella) [inline examples/scripts/compare_lithium_ion.py:13]: I think we should restore this script to its previous version.

**Review by brosaplanella — APPROVED** [review]
Looks good, just a question: this is a dimensionless surface area to volume so it is `1` when there is no degradation, right?

File: examples/scripts/compare_lithium_ion.py (hunk 1 above)
Comment (tinosulzer) [reply examples/scripts/compare_lithium_ion.py:13]: oops yes

**Comment by tinosulzer** [comment]
Yes it's dimensionless

**Pull request merged**